pkg/arvo/sys/zuse.hoon

```diff
@@ -2357,6 +2357,149 @@
   ++  blake
     ~%  %blake  ..part  ~
     |%
+    ++  blake3
+      =<
+        =<  hash  :: cuter API
+        =+  [cv=iv flags=0b0]
+        ^?  ~%  %blake3  ..output  ~
+        |%
+        ::
+        ++  keyed  |=(key=octs hash(cv q.key, flags f-keyedhash))
+        ::
+        ++  hash
+          ~%  %hash  ..hash  ~
```

Comment:
@joemfb can these be simplified to `~/  %hash` now that the parents are fixed?

Reply:
Sorry for the delay. Yes, all the actual jets here can be `~/`, as can this `%blake3` core (but not `%blake3-impl`).